test/iavl_test.go

```diff
@@ -6,9 +6,11 @@ import (
 	"testing"
 
 	"github.com/ethereum/go-ethereum/common"
+	qt "github.com/frankban/quicktest"
```

Comment:
This file (`test/iavl_test.go`) is no longer testing anything related to `iavl`, maybe it can be renamed to a more up-to-date filename?